analyze_test.go

```diff
@@ -119,6 +119,50 @@ func TestAnalyze_auth_jwt(t *testing.T) {
 	}
 }
 
+func TestAnalyze_auth_jwt_require(t *testing.T) {
+	t.Parallel()
+	testcases := map[string]struct {
```

Comment:
For completeness should there be a test to cover `auth_jwt_require $var error=403`?

Reply:
done, added a test for the error code